src/FormBuilder/resources/js/form-builder/src/settings/group-email-settings/index.tsx

```diff
@@ -0,0 +1,27 @@
+import {getFormBuilderWindowData} from '@givewp/form-builder/common/getWindowData';
+import EmailTemplateOptions from './email/template-options';
+import {useFormState} from '@givewp/form-builder/stores/form-state';
+
+const getEmailSettings = () => {
+    const {emailNotifications} = getFormBuilderWindowData();
+    const {
+        settings: {emailOptionsStatus},
+    } = useFormState();
+    let emailSettings = [];
+
+    if (emailOptionsStatus === 'enabled') {
+        emailSettings = emailNotifications.map((emailNotification) => {
+            const {id, title} = emailNotification;
+
+            return {
+                name: title,
+                path: `email-settings/${id}`,
+                element: <EmailTemplateOptions notification={id} />,
+            };
+        });
+    }
+
+    return emailSettings;
+}
```

Comment:
Hmm, could be interesting to have a `showWhen` property of the route object that accepts a callback which returns true/false. That may have some rendering complications though, so not a big deal. Just a thought. 😄